src/DependencyInjection/Configuration.php

```diff
@@ -59,6 +59,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                                 ->info('Property accessor path (like method or property name) used to decide if an entry should be indexed.')
                                 ->defaultNull()
                             ->end()
+                            ->scalarNode('data_provider')
+                                ->info('Method of the entity repository called when the meilisearch:import command is invoked.')
+                                ->defaultNull()
+                            ->end()
```

Comment:
I think we should introduce something like:
```
persistence:
    driver: orm # later odm, custom (default to orm, to not introduce BC break immediately)
    data_provider: null # if set, use it, otherwise register the default orm provider and add it to service locator
```